core/Dockerfile

```diff
@@ -30,4 +30,4 @@ COPY . /app
 
 WORKDIR /app
 
-ENTRYPOINT ["uvicorn", "main:app", "--proxy-headers"]
+CMD alembic upgrade head && uvicorn main:app --proxy-headers 
```

Comment:
Weird why? ENTRYPOINT makes sense to me, but we can also do a CMD here. Why did you change that?

Reply:
Alternative would be `ENTRYPOINT ["sh", "-c", "alembic upgrade head &&  uvicorn main:app --proxy-headers"]`, or am I missing something? Seemed more natural using CMD here.